app/services/populate_issues.rb

```diff
@@ -0,0 +1,30 @@
+class PopulateIssues
```

Comment:
This code is only used by the PopulateIssuesJob right? As in, it isn't used anywhere else?

If so, I don't see the point of making it a separate class when PopulateIssuesJob just calls PopulateIssues.call. Probably should just move all the code into PopulateIssuesJob.